Formula/naabu.rb

```diff
@@ -1,9 +1,9 @@
 class Naabu < Formula
   desc "Fast port scanner"
   homepage "https://github.com/projectdiscovery/naabu"
-  url "https://github.com/projectdiscovery/naabu/archive/v1.1.4.tar.gz"
-  sha256 "fb5fae5ef7ef45217a238b59ed16095960c197c6f8b2f460873a445dbc80c2b4"
-  license "GPL-3.0"
+  url "https://github.com/projectdiscovery/naabu/archive/v2.0.0.tar.gz"
+  sha256 "0d9e970c70b10d56695c8836847ee91a6717ceb1054e1ac7eb1d1c01437e0aa0"
+  license "GPL-3.0-only"
```

Comment:
if we can revert the license change, i think we can ship out the change.

Reply:
cc @Mzack9999

Reply:
@chenrui333 The license has become MIT (https://github.com/projectdiscovery/naabu/blob/master/LICENSE.md)